.github/workflows/nightly.yml

```diff
@@ -0,0 +1,21 @@
+name: Run compiler tests
+
+on:
+  schedule:
+    - cron: "0 6 * * *"  # Run daily at 06:00 UTC
+  workflow_dispatch:     # Allow manual dispatching
+  pull_request:
+
+jobs:
+  compile-test:
+    name: Rust Compiler Tests
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
```

Comment:
```suggestion
        uses: actions/checkout@v3
        uses: dtolnay/rust-toolchain@nightly
```

Reply:
This is downloading the project-stable-mir repo not rust.